django/contrib/admin/options.py

```diff
@@ -1583,6 +1584,24 @@ def add_view(self, request, form_url='', extra_context=None):
     def change_view(self, request, object_id, form_url='', extra_context=None):
         return self.changeform_view(request, object_id, form_url, extra_context)
 
+    def get_edited_object_pks(self, request, prefix):
```

Comment:
I would organize these methods else (inside of in the middle of the views) and prefix them with an underscore to indicate that they're helpers, not public APIs.

Reply:
So they should be defined inside `changelist_view`?